.github/workflows/ci.yml

```diff
@@ -17,8 +17,8 @@ env:
   CACHE_NUMBER: 1
 
 jobs:
-  unit-test-frontend:
-    name: Frontend linting / unit tests
+  unit-test-frontend2:
```

Comment:
Are you planning to rename this to frontend eventually when the entire branch gets merged?

(at some point it becomes safe to delete the frontend directory entirely?)

Reply:
We're planning on merging frontend2 branch into main before onboarding people, so we'll keep things in the frontend2 folder for now. We can rename the tests and everything else when frontend2 is ready to be the frontend